scripts/install_narrative_docker.sh

```diff
@@ -8,12 +8,10 @@
 PYTHON=python
 
 SCRIPT_TGT="kbase-narrative"
-SCRIPT_TGT2="headless-narrative"
```

Comment:
this script exists but it points at a python file that no longer does ==> delete the code so that people aren't tempted to try it